app/post.go

```diff
@@ -251,7 +251,7 @@ func (a *App) CreatePost(c request.CTX, post *model.Post, channel *model.Channel
 			post.AddProp("attachments", attachmentsInterface)
 		}
 		if err != nil {
-			mlog.Warn("Could not convert post attachments to map interface.", mlog.Err(err))
+			a.Log().Warn("Could not convert post attachments to map interface.", mlog.Err(err))
```

Comment:
`c.Logger().Warn`

Reply:
Nice catch. I missed the context argument.